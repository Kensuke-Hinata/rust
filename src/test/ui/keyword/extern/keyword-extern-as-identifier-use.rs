use extern::foo; //~ ERROR expected identifier, found keyword `extern`
                 //~| ERROR unresolved import `extern`

fn main() {}
