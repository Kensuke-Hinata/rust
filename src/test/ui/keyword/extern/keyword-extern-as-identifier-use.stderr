error: expected identifier, found keyword `extern`
  --> $DIR/keyword-extern-as-identifier-use.rs:1:5
   |
LL | use extern::foo; //~ ERROR expected identifier, found keyword `extern`
   |     ^^^^^^ expected identifier, found keyword
help: you can escape reserved keywords to use them as identifiers
   |
LL | use r#extern::foo; //~ ERROR expected identifier, found keyword `extern`
   |     ^^^^^^^^

error[E0432]: unresolved import `extern`
  --> $DIR/keyword-extern-as-identifier-use.rs:1:5
   |
LL | use extern::foo; //~ ERROR expected identifier, found keyword `extern`
   |     ^^^^^^ could not find `extern` in `{{root}}`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
